### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/lynis/commit/0b4158fee3c0936d8a2c5ec52e5931d8674edb55#commitcomment-43995442

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/lynis/commit/da45f48c3f06a1341f6062ac5ed406df58f2d91d

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3189,11 +3189,11 @@ Lynis 1.1.2 (2008-05-11)
  - Improved LILO test and removed double message
  - Fixed incorrect message when using --help parameter
  - Improved portaudit test (FreeBSD) to show unique packages only
- - Updated man page, FAQ, extended documention with plugin information
+ - Updated man page, FAQ, extended documentation with plugin information
  - Added several php.ini file locations (MacOS X, OpenBSD, OpenSuSE)
 
  ** Special release notes [package/ports]: **
- - Added several default paths to check for usuable an INCLUDE directory. This
+ - Added several default paths to check for usable INCLUDE directory. This
```
Comment (jsoref) [inline CHANGELOG.md:3196]: I'm not certain about this

File: CHANGELOG.md [hunk 2]
```diff
@@ -3465,7 +3465,7 @@ Lynis 1.0.0 (2007-11-08)
  - Test: query nameservers and test connectivity
  - Test: check promiscuous interfaces (FreeBSD)
  - Test: check sticky bit on /tmp directory
- - Test: check debian.org security brance in /etc/apt/sources.list
+ - Test: check debian.org security branch in /etc/apt/sources.list
```
Comment (jsoref) [inline CHANGELOG.md:3468]: I'm not certain about this

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -68,6 +68,6 @@ software or computer software documentation in whole or in part, in any manner
 and for any purpose whatsoever, and to have or authorize others to do so.
 
 If you want to be named in as a contributor in the CONTRIBUTOR file, then include
-this notition in your pull request. Preferred format: Full Name, and your e-mail
+this notation in your pull request. Preferred format: Full Name, and your e-mail
```
Comment (jsoref) [inline CONTRIBUTING.md:71]: I'm not certain about this

File: extras/bash_completion.d/lynis [hunk 4]
```diff
@@ -126,7 +126,7 @@ _lynis()
         report)
             return 0
             ;;
-        settiings)
+        settings)
```
Comment (jsoref) [inline extras/bash_completion.d/lynis:129]: this is scary

File: include/helper_configure [hunk 5]
```diff
@@ -72,7 +72,7 @@
             ExitFatal
         fi
 
-        FIND=$(echo ${HELPER_PARAMERS} | grep " ")
+        FIND=$(echo ${HELPER_PARAMS} | grep " ")
```
Comment (jsoref) [inline include/helper_configure:75]: this is scary

File: include/profiles [hunk 6]
```diff
@@ -50,7 +50,7 @@
             Display --text " "
             Display --text "=================================================================================================="
             Display --text " "
-            LogText "Insight: Profile '${PROFILE}' contians one or more old-style configuration entries"
+            LogText "Insight: Profile '${PROFILE}' contains one or more old-style configuration entries"
```
Comment (jsoref) [inline include/profiles:53]: this is presumably a log output change

File: include/report [hunk 7]
```diff
@@ -151,14 +151,14 @@
             fi
 
             # Show suggestions from logfile
-            SSUGGESTIONS=$(${GREPBINARY} 'Suggestion: ' ${LOGFILE} | sed 's/ /!space!/g')
+            SUGGESTIONS=$(${GREPBINARY} 'Suggestion: ' ${LOGFILE} | sed 's/ /!space!/g')
```
Comment (jsoref) [inline include/report:154]: this seems harmless

File: include/tests_kernel_hardening [hunk 8]
```diff
@@ -28,7 +28,7 @@
 #
     # Test        : KRNL-6000
     # Description : Check sysctl parameters
-    # Sysctl      : net.ipv4.icmp_ingore_bogus_error_responses (=1)
+    # Sysctl      : net.ipv4.icmp_ignore_bogus_error_responses (=1)
```
Comment (jsoref) [inline include/tests_kernel_hardening:31]: there's pretty much only one other instance of this typo in the world

**Comment by mboelen** [comment]
Thanks, merged!

**Pull request merged**